### DISCUSSION TITLE
install fastMRI

### DISCUSSION THREAD

**Discussion by ZhengguoTan** [facebookresearch/fastMRI#242] [General]
Dear fastMRI developers,

I may have a question with regard to the installation of the fastMRI toolbox.

I firstly create and activate a conda environment:

$ conda create -n fastMRI
$ conda activate fastMRI

Then I install python and pytorch:

$ conda install -c anaconda python
$ conda install pytorch torchvision torchaudio cudatoolkit=10.2 -c pytorch

Afterwards, I try to install the requirements:

$ pip install -r requirements.txt

which fails. 

In this case, I may need to install all the required packages in requirements.txt one by one. 

May you know why the installation using requirements.txt does not work? It would be great if you could help to resolve this problem.

Best Regards,
Zhengguo

**Comment by mmuckley** [comment]
Hello @ZhengguoTan, thanks for your question.

Unfortunately this is mostly a product of bad naming - the file is named `requirements.txt` but it is not actually requirements, it its a bunch of fixed package versions we use for testing the code in our continuous integration test suite. It is not meant to specify requirements for the end use, but if you encounter any issues you can fall back to these versions for debugging.

All this is to say is that you should be able to ignore `requirements.txt` and install via `pip install -e .` if installing from source or `pip install fastmri` if you're using PyPI.
